page.codeberg.tpikonen.satellite.appdata.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally, you should maintain this file in your upstream repo. It's not specific to Flathub, and you can also update it when making a new release.

Reply:
I removed the appdata, it's indeed already in the sources.